app/login/page.jsx

```diff
@@ -65,50 +65,91 @@ export default function Login() {
     }
   }
 
-  return (
-    <main className="flex min-h-screen flex-col items-center">
-      <div className="pt-20">
-        <LogoBig />
-      </div>
-      <form>
-        <div className="relative mb-4">
-          <input
-            onChange={e => setEmail(e.target.value)}
-            type="email"
-            id="email"
-            placeholder="Email"
-            className="mt-16 w-full rounded border-2 border-neutral-400 p-2"
-          />
-          <label htmlFor="email" className="sr-only">
-            Email
-          </label>
-          <input
-            onChange={e => setPassword(e.target.value)}
-            type="password"
-            id="password"
-            placeholder="Password"
-            className="my-2 w-full rounded border-2 border-neutral-400 p-2"
-          />
-          <label htmlFor="password" className="sr-only">
-            Password
-          </label>
-          <button className="w-full text-right text-sm font-semibold text-neutral-700">
-            Forgot password?
-          </button>
-        </div>
-        <div className="mb-12 flex flex-col pb-1 pt-1 text-center">
-          <button
-            onClick={e => handleOnSubmit(e)}
-            className="mb-2 rounded bg-neutral-900 p-4 font-semibold text-neutral-100">
-            Log in
-          </button>
-          <button type="button">
-            <p className="mb-0 mr-2 text-sm font-semibold text-neutral-700">
-              Don&apos;t have an account? Register
-            </p>
-          </button>
-        </div>
-      </form>
-    </main>
-  )
+  const switchToView = newView => {
+    setEmail('')
+    setPassword('')
+    setView(newView)
+  }
+
+   return (
+     <main className="flex min-h-screen flex-col items-center">
+       <div className="pt-20">
+         <LogoBig />
+       </div>
+       <form>
+         <div className="relative mb-4">
+           {view === 'reset-password' ? (
+             // Display only email input field for reset-password view
+             <input
```

Comment:
does this input really have to be copy pasted? i think you could write a condition that works with one